### PR TITLE
Fix build, add example app, remove bitflags

### PR DESCRIPTION
**Pull request by carbotaniuman**
**What kind of change does this PR introduce?** (check at least one)

- [X] Bugfix
- [X] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [X] No

**The PR fulfills these requirements:**

- [X] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: src/sample.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+use std::ptr::null_mut;
```
Comment (nothingismagick) [inline src/sample.rs]: Shouldn't we put this in `examples/` or `test/` ?

**Review by nothingismagick — COMMENTED** [review]
Not sure about `sample.rs` WDYT?

File: src/sample.rs (hunk 1 above)
Comment (carbotaniuman) [reply src/sample.rs]: The samples is just here for testing, I'll prob put it in an example folder.

**Review by khionu — CHANGES_REQUESTED** [review]
`build.rs` doesn't have the rerun-if lines. This is necessary for when any changes happen to the C code.

Should just have something like this near the beginning.
`println!("cargo:rerun-if-changed=webview.h");`

File: src/sample.rs (hunk 1 above)
Comment (tensor-programming) [reply src/sample.rs]: Aye, its just to test the example.

**Pull request merged**